vanilla_installer/defaults/users.py

```diff
@@ -95,10 +95,10 @@ def __on_username_entry_changed(self, *args):
             _status = False
             self.__window.toast("Username cannot be empty. Please type a username.")
 
-        # cannot be root
-        elif _input == "root":
+        # cannot be root, ubuntu, vanillaos or vanilla-os
+        elif _input == "root" or _input == "vanillaos" or _input == "vanilla-os":
```

Comment:
```suggestion
        # cannot be root, debian, vanillaos or vanilla-os
        elif _input == "root" or _input == "vanillaos" or _input == "vanilla-os" or _input=='debian':
```
Thanks for your contribution, This suggestion is for now, I think Mirkob's suggested approach would be better in the long term.